ui/src/app/features/table/table.component.html

```diff
@@ -22,6 +22,6 @@
   </ng-container>
 
 
-  <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+  <tr mat-header-row *matHeaderRowDef="displayedColumns; sticky: true"></tr>
```

Comment:
should it be just sticky? or sticky="true"

Reply:
sticky="true" .